### PR TITLE
[HRIS - 447] - [BUGTASK] The Type of Leave is not displayed in the approved or disapproved notification modal

### PR DESCRIPTION
**Pull request by jessrey-asterisk**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/view/HRIS-447)

## Definition of Done

- [x] Users can see the Type of Leave in the Resolved Notifications for Leave

## Pre-condition

Commands to run
npm run dev
dotnet run

## Expected Output

it should display the type of leave in approved and disapproved notification windows

## Screenshots/Recordings

[Disapproved](https://drive.google.com/file/d/1ApvliXJt9LupVmennyXtVODSNIE6sk2C/view?usp=sharing)

[Approved](https://drive.google.com/file/d/18i68pK9Z9i-C5O2S5LrWpq9C0gSEHav2/view?usp=sharing)

### CODE REVIEW

**Review by Fontilllas — COMMENTED** [review]
-After Approve or Disapproving the Leave Request the type of Leave does not display please resolve this one.
-User Type: Employee, Project Leader, Manager


Approve:

https://github.com/user-attachments/assets/0a3d68b8-b412-4c71-9716-7344d23996f2

Disapprove:

https://github.com/user-attachments/assets/4f66b39c-f7ba-432b-a3f3-358f0ac33582


Comments: 1
Testing Time: 20min
Iterations: 0

**Comment by JohnlynDC** [comment]
Disapproved

- The user cannot see the type of leave in the approved and disapproved notification modal.
- Please resolve this one.

Testing for Approve:

https://github.com/user-attachments/assets/b8f85592-2a3d-4ce3-9e19-829ee9a34f55

Testing Disapprove:

https://github.com/user-attachments/assets/fd43e311-03fa-4d83-919f-c7b1b54e892d

Comments: 2
Testing Time: 20mins
Iterations: 1

**Review by jonel371234 — APPROVED** [review]
Approved
- Users can see the Type of Leave in the Resolved Notifications for Leave 
- Pending Status
- Approved by both admin and manager
- Disapproved by both admin and manager


https://github.com/user-attachments/assets/d73b2046-cb53-4d92-9d8b-8a1d4ee09445

Comments: When I checked out to the branch, there were some changes that didn’t reflect on my end, which is why the type of leave was not displayed. However, Sir Jessrey taught me how to fix it, and now it’s all good
Test Time: 10mins
Iterations: 0

**Review by Miguel21Monacillo — APPROVED** [review]
-[x] After checking request, the type of leave can now be viewed for both approve and disapprove

- Approve

https://github.com/user-attachments/assets/ca1084c2-d874-4e7d-92e5-dce2ead58441

- Disapprove

https://github.com/user-attachments/assets/1e324a8d-989e-4624-8dc4-ea6a44873902

File: sph-hris.sln
Comment (Miguel21Monacillo) [inline sph-hris.sln:1]: There is a conflict in this file, please fix

**Review by Miguel21Monacillo — COMMENTED** [review]
Please fix conflict

File: sph-hris.sln
Comment (jessrey-asterisk) [reply sph-hris.sln:1]: Done

**Review by Fontilllas — APPROVED** [review]

**Pull request merged**